### PR TITLE
Update cephfs and ceph rgw related documents

### LINKED ISSUE #13515: Update cephfs related documents.
**Page**

https://docs.alluxio.io/os/user/stable/en/ufs/Ceph.html
https://docs.alluxio.io/os/user/stable/cn/ufs/Ceph.html

**Summary**

Refactor the document to let user know how to use ceph or cephfs as ufs of alluxio.

### PR DESCRIPTION
**Pull request by jhonxue**
Update ufs cephfs related documents

Fix #13515

Signed-off-by: Xue Yantao xueyantao2114@163.com

### What changes are proposed in this pull request?

Please outline the changes and how this PR fixes the issue.

### Why are the changes needed?

Please clarify why the changes are needed. For instance,
  1. If you propose a new API, clarify the use case for a new API.
  2. If you fix a bug, describe the bug.

### Does this PR introduce any user facing changes?

Please list the user-facing changes introduced by your change, including
  1. change in user-facing APIs
  2. addition or removal of property keys
  3. webui

### CODE REVIEW

File: docs/cn/ufs/CephFS.md [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+该指南介绍如何配置Alluxio以使用CephFS作为底层文件系统。Alluxio使用[CephFS](https://docs.ceph.com/en/latest/cephfs/)
+目前，支持两种不同类型的UFS CephFS：
+- [cephfs](https://docs.ceph.com/en/latest/cephfs/api/libcephfs-java/)
+- [cephfs-hadoop](https://docs.ceph.com/en/nautilus/cephfs/hadoop/)
+
+## 初始步骤
+
+首先，在你的机器上必须安装Alluxio二进制包。你可以自己[编译Alluxio](Building-Alluxio-From-Source.html)，或者[下载二进制包](Running-Alluxio-Locally.html)
+
+其次，在你的机器上需要安装一下包:
```
Comment (apc999) [inline docs/cn/ufs/CephFS.md:21]:
```suggestion
其次，在你的机器上需要安装以下包:
```

File: docs/en/ufs/CephFS.md [hunk 2]
```diff
@@ -0,0 +1,192 @@
+---
+layout: global
+title: CephFS
```
Comment (apc999) [inline docs/en/ufs/CephFS.md:3]: can you modify https://github.com/Alluxio/alluxio/blob/master/docs/en/ufs/Ceph.md to be CephObjectStore to differentiate from CephFS?

File: docs/en/ufs/CephFS.md [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+nickname: CephFS
+group: Storage Integrations
+priority: 11
+---
+
+* Table of Contents
+{:toc}
+
+This guide describes how to configure Alluxio with CephFS as the under storage system. Alluxio supports
+two different under storage systems to connect to [CephFS](https://docs.ceph.com/en/latest/cephfs/):
```
Comment (apc999) [inline docs/en/ufs/CephFS.md:13]:
```suggestion
two different implementations of under storage system for [CephFS](https://docs.ceph.com/en/latest/cephfs/):
```

File: docs/en/ufs/CephFS.md [hunk 4]
```diff
@@ -0,0 +1,192 @@
...
+### Deploy Alluxio binary package
+The Alluxio binaries must be on your machine. You can either
+[compile Alluxio]({{ '/en/contributor/Building-Alluxio-From-Source.html' | relativize_url }}), or
+[download the binaries locally]({{ '/en/deploy/Running-Alluxio-Locally.html' | relativize_url }}).
+
+### Install Dependences
+According to [ceph packages install](https://docs.ceph.com/en/latest/install/get-packages/)，
+    cephfs-java
+    libcephfs_jni
+    libcephfs2
```
Comment (apc999) [inline docs/en/ufs/CephFS.md:28]: unformatted

File: docs/en/ufs/CephFS.md [hunk 5]
```diff
@@ -0,0 +1,192 @@
...
+Configure Alluxio to use under storage systems by modifying
+`conf/alluxio-site.properties` and `conf/core-site.xml`. If them do not exist, 
+create the configuration files from the templates
+
+```console
+$ cp conf/alluxio-site.properties.template conf/alluxio-site.properties
+$ cp conf/core-site.xml.template conf/core-site.xml
+```
+
+### Option 1: ufs cephfs
```
Comment (apc999) [inline docs/en/ufs/CephFS.md:56]: checkout the new way of doing this to implement tabs:

```
{% navtabs Setup %}
{% navtab Option 1: cephfs %}
....
{% endnavtab %}
{% navtab Option 2: cephfs-hadoop %}
...
{% endnavtab %}
{% endnavtabs %}
```

See how it looks like at [this page](https://docs.alluxio.io/os/user/stable/en/deploy/Running-Alluxio-On-Docker.html#launch-alluxio-containers-for-master-and-worker)

File: docs/en/ufs/CephFS.md [hunk 6]
```diff
@@ -0,0 +1,192 @@
...
+
+Modify `conf/alluxio-site.properties` to include:
+
+```properties
+alluxio.underfs.cephfs.conf.file=<ceph-conf-file->
```
Comment (apc999) [inline docs/en/ufs/CephFS.md:61]:
```suggestion
alluxio.underfs.cephfs.conf.file=<ceph-conf-file>
```

File: docs/en/ufs/CephFS.md [hunk 7]
```diff
@@ -0,0 +1,192 @@
...
+
+```console
+$ ./bin/alluxio format
+$ ./bin/alluxio-start.sh local
+```
+
+This should start an Alluxio master and Alluxio worker. You can see the master UI at
+[http://localhost:19999](http://localhost:19999).
+
+### Option 1: ufs cephfs
```
Comment (apc999) [inline docs/en/ufs/CephFS.md:134]: Same here, let's use tabs

```
{% navtabs Examples %}
{% navtab Option 1: cephfs %}
....
{% endnavtab %}
{% navtab Option 2: cephfs-hadoop %}
...
{% endnavtab %}
{% endnavtabs %}
```

File: docs/en/ufs/CephFS.md [hunk 8]
```diff
@@ -0,0 +1,192 @@
...
+In cephfs, you can visit cephfs with ceph-fuse or mount by POSIX APIs. [Mounting CephFS](https://docs.ceph.com/en/latest/cephfs/#mounting-cephfs)
+```
+${ceph-fs-dir}/default_tests_files/Basic_CACHE_THROUGH
+```
+In Alluxio, you can visit the nested directory in the Alluxio. Alluxio's [Command Line Interface]({{ '/en/operation/User-CLI.html' | relativize_url }}) can be used for this purpose.
+```
+/mnt/cephfs/default_tests_files/Basic_CACHE_THROUGH
+```
+
+
```
Comment (apc999) [inline docs/en/ufs/CephFS.md:163]: unnecessary emptyline

File: docs/en/ufs/CephFS.md [hunk 9]
```diff
@@ -0,0 +1,192 @@
...
+### Option 2: ufs cephfs-hadoop
+
+An CephFS location can be mounted at a nested directory in the Alluxio namespace to have unified access
+to multiple under storage systems. Alluxio's [Command Line Interface]({{ '/en/operation/User-CLI.html' | relativize_url }}) can be used for this purpose.
+
+Issue the following command to use the ufs cephfs:
+```console
+./bin/alluxio fs mkdir /mnt/cephfs-hadoop
+./bin/alluxio fs mount /mnt/cephfs-hadoop ceph://mon1\;mon2\;mon3/
```
Comment (apc999) [inline docs/en/ufs/CephFS.md:172]:
```suggestion
$ ./bin/alluxio fs mkdir /mnt/cephfs-hadoop
$ ./bin/alluxio fs mount /mnt/cephfs-hadoop ceph://mon1\;mon2\;mon3/
```

**Review by apc999 — CHANGES_REQUESTED** [review]

File: docs/en/ufs/CephFS.md (hunk 2 above)
Comment (jhonxue) [reply docs/en/ufs/CephFS.md:3]: > can you modify https://github.com/Alluxio/alluxio/blob/master/docs/en/ufs/Ceph.md to be CephObjectStore to differentiate from CephFS?

yes, i am glad to do it.

**Comment by jhonxue** [comment]
@apc999 PTAL

**Comment by jhonxue** [comment]
@apc999 PTAL

**Review by apc999 — APPROVED** [review]

**Comment by apc999** [comment]
alluxio-bot, merge this please

**Pull request merged**